data/korath ships.txt

```diff
@@ -955,6 +968,23 @@ ship "Model 16"
 	description "The Kor Mereti war drones are constructed in a modular fashion, which allows them to grow over time as more materials become available. This one is recently built, and relatively small."
 
 
+ship "Model 16" "Model 16 (Jump Miner)"
```

Comment:
I edited my comment after that. My question is, why is Model 8's just "Jump" when 16's is "Jump Miner"? Why are they different?